enterprise/server/invocation_stat_service/invocation_stat_service.go

```diff
@@ -556,6 +557,19 @@ func (i *InvocationStatService) getMetricBuckets(ctx context.Context, table stri
 	return metricBuckets, metricArrayStr, nil
 }
 
+func getTimeIncrement(duration time.Duration) time.Duration {
```

Comment:
little comment here would be nice for the non-drilldown-initiated

Reply:
Fixed the function and flag name to suck a little less, comment is hopefully mostly redundant now but added it anyway.